apps/drain-safe/src/utils/chains.ts

```diff
@@ -0,0 +1,31 @@
+export enum CHAINS {
+  MAINNET = '1',
+  MORDEN = '2',
+  ROPSTEN = '3',
+  RINKEBY = '4',
+  GOERLI = '5',
+  KOVAN = '42',
+  BSC = '56',
+  XDAI = '100',
+  POLYGON = '137',
+  ENERGY_WEB_CHAIN = '246',
+  ARBITRUM = '42161',
+  VOLTA = '73799',
+}
+
+export const rpcUrlGetterByNetwork: {
+  [key in CHAINS]: null | ((token?: string) => string);
+} = {
+  [CHAINS.MAINNET]: (token) => `https://mainnet.infura.io/v3/${token}`,
+  [CHAINS.MORDEN]: null,
```

Comment:
Do we want to include chains to this enum if we don't support them?

Reply:
We are doing a copy 1:1 of what we have on TX-Builder. This is to be deprecated with [#235](https://github.com/gnosis/safe-react-apps/issues/235) in both apps